tests/integration/golang/aim/run/search_test.go

```diff
@@ -237,6 +241,24 @@ func (s *SearchTestSuite) Test_Ok() {
 		request request.SearchRunsRequest
 		runs    []*models.Run
 	}{
+		{
+			name: "SearchSecondPage",
+			request: request.SearchRunsRequest{
+				Query:  `run.archived == True or run.archived == False`,
```

Comment:
this could actually be `run.archived or not run.archived` — generally speaking this `run.archived == True` syntax is not very Pythonic

Reply:
this is want we have in other tests. so I prefer to keep it as is and changed it later everywhere.

Reply:
agreed — let's change this in another PR